TYPO3.Flow/Classes/TYPO3/Flow/Core/ClassLoader.php

```diff
@@ -440,7 +440,9 @@ protected function initializeAutoloadInformation($composerPath, ApplicationConte
         }
 
         if ($context !== null) {
-            $proxyClasses = @include(FLOW_PATH_DATA . 'Temporary/' . (string)$context . '/AvailableProxyClasses.php');
+            // see Environment::createTemporaryDirectory()
+            $temporaryDirectory = FLOW_PATH_DATA . 'Temporary/' . str_replace('/', '/SubContext', (string)$context) . '/';
```

Comment:
Not necessarily something to fix in this change but as I see it, shouldn't Context have a getConfigurationPath() method or something like that... This string manipulation stuff with hardcoded strings makes me edgy ;)